libcxx/docs/Modules.rst

```diff
@@ -214,6 +219,62 @@ Building this project is done with the following steps, assuming the files
 
              ``error: module file _deps/std-build/CMakeFiles/std.dir/std.pcm cannot be loaded due to a configuration mismatch with the current compilation [-Wmodule-file-config-mismatch]``
 
+
+Using the installed modules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CMake has added experimental support for importing the standard modules. This
+is available in the current nightly builds and will be part of the 3.30
+release. Currently CMake only supports importing the Standard modules in C++23
```

Comment:
Do we want to write "Standard" or "standard"?

Reply:
The official wording uses `International Standard` so I tend to capitalize; that is if I don't forget to do it ;-)

Reply:
I thought so -- I am terrible with that sort of thing :-)